.travis.yml

```diff
@@ -3,11 +3,9 @@ osx_image: xcode8.3
 
 before_script:
   - bundle install
-  - cd Bootstrap ; bundle exec pod install --repo-update ; cd ..
+  - cd Bootstrap ; bundle exec pod install ; cd ..
```

Comment:
It's possible, but we can add it back in later if we need to. I took it out to save time on CI, now I'm not sure it's worth it. Any thoughts?